src/popup.html

```diff
@@ -8,8 +8,21 @@
 <body>
   <div class="card">
     <div class="card-header bg-dark text-white py-1 d-flex align-items-center">
-      <img class="logo" alt="CQG Inc. logo" src="./img/cqglogo.svg">
-      <div class="ml-auto" id="timer-counter">00:00:00</div>
+      <div class="container">
+        <div class="row">
+          <div class="col align-self-center">
+            <img class="logo" alt="CQG Inc. logo" src="./img/cqglogo.svg">
+          </div>
+          <div class="col-auto">
+            <div class="ml-auto" id="timerCounter">00:00:00</div>
```

Comment:
Let's have classes here instead of ids. Say 
`<div class="ml-auto timer-counter">00:00:00</div>`
and
`<div class="ml-auto text-right timer-counter timer-counter-total">00:00:00</div>`

If so we can inherit some properties in css:
```
.timer-counter {
  font-size: 14px;
  line-height: 1;
}
.timer-counter-total {
  color: grey;
}
```

Ids can be left for selecting elements from DOM.